SimpleWAWS/Code/BackgroundQueueManager.cs

```diff
@@ -61,26 +63,31 @@ public void DeleteResourceGroup(ResourceGroup resourceGroup)
 
         private async Task<ResourceGroup> LogActiveUsageStatistics(ResourceGroup resourceGroup)
         {
-            try
+            if (resourceGroup.SubscriptionType == Subscription.SubscriptionType.AppService)
             {
-                var site = resourceGroup.Sites.First(s => s.IsSimpleWAWSOriginalSite);
-                var credentials = new NetworkCredential(site.PublishingUserName, site.PublishingPassword);
-                var zipManager = new RemoteZipManager(site.ScmUrl + "zip/", credentials);
-                using (var httpContentStream = await zipManager.GetZipFileStreamAsync("LogFiles/http/RawLogs"))
+                try
                 {
-                    await StorageHelper.UploadBlob(resourceGroup.ResourceUniqueId, httpContentStream);
+                    foreach (var site in resourceGroup.Sites)
```

Comment:
`StorageHelper.UploadBlob` and `StorageHelper.AddQueueMessage` are called in a for loop, with the same uniqueId which is the name of the blob